static/css/global.css

```
@@ -185,6 +185,26 @@ footer {
   opacity: 0.5;
 }
 
+.pagination-number-button {
+  background-color: #236BE4;
+  border-color: #236BE4;
+  color: #fff;
+  min-width: 2.5em;
+  font-size: 1em;
+  justify-content: center;
+  margin: .25rem;
+  padding-left: .5em;
+  padding-right: .5em;
```

@MeliAlbu tanto para margin como padding, cuando tenés las 4 propiedades, conviene simplificarlas en un shorthand, quedaría así:
```suggestion
padding: calc(.5em - 1px) .5em .5em;
```